spring-thymeleaf-2/src/main/resources/templates/mvcdata/email-bean-data.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+	  xmlns:th="http://www.thymeleaf.org">
```

Comment:
To adhere to the Baeldung style guide, let's please have this be a 2-space indent. Or it can go onto the previous line.